cpp/src/io/parquet/page_data.cu

```diff
@@ -426,7 +426,7 @@ __global__ void __launch_bounds__(decode_block_size)
                     device_span<ColumnChunkDesc const> chunks,
                     size_t min_row,
                     size_t num_rows,
-                    int32_t* error_code)
+                    kernel_error::pointer error_code)
```

Comment:
This looks awkward IMO. I feel much better with `kernel_error::value_type*`.

Reply:
I think it's fine. `pointer` is a pretty standard naming.

Reply:
And is consistent with the `device_scalar` interface, as @vuule pointed out above.